Kickstarter-iOS/Views/Controllers/DashboardViewController.swift

```diff
@@ -158,7 +176,7 @@ internal final class DashboardViewController: UITableViewController {
     }
   }
 
-  fileprivate func goToMessages(_ project: Project) {
+  fileprivate func goToMessages(project: Project) {
```

Comment:
I think that at some point we can refactor all of our `fileprivate`s to `private`s where it makes sense. If you'd like to change all of them here that's cool but I think ideally we just don't want to create _more_ `fileprivate`s.